simulations/vip-502/bscmainnet.ts

```diff
@@ -0,0 +1,20 @@
+import { expectEvents } from "../../src/utils";
+import { forking, testVip } from "../../src/vip-framework";
+import vip408 from "../../vips/vip-408/bscmainnet";
+import vip502 from "../../vips/vip-502/bscmainnet";
+import OMNICHAIN_PROPOSAL_SENDER_ABI from "../vip-502/abi/OmnichainProposalSender.json";
+
+forking(44978531, async () => {
+  testVip("VIP-408 Multichain Governance", await vip408(), {});
```

Comment:
VIP-408 has been already executed. We should remove this and update the block number